tests/unit/package_managers/python/pip/test_markers.py

```
@@ -0,0 +1,517 @@
+    def test_comparisons_raise(self, op: str) -> None:
+        import operator
+
+        poison = _Poison("platform_machine")
+        with pytest.raises(_UncontrolledEnvKey) as exc_info:
+            getattr(operator, op)(poison, "x86_64")
+        assert exc_info.value.key == "platform_machine"
+
+    def test_membership_raises(self) -> None:
+        poison = _Poison("platform_release")
+        with pytest.raises(_UncontrolledEnvKey) as exc_info:
+            "x" in poison
```

Ruff (0.16.1)

[warning] Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)